src/components/templates/AvatarGrid/AvatarGrid.tsx

```diff
@@ -382,12 +381,9 @@ const AvatarGrid = () => {
       >
         <div className="grid-rooms-container">
           {venue.spaces?.map((room: AvatarGridRoom, index: number) => {
-            const peopleInRoom = partygoers
-              ? partygoers.filter(
-                  (partygoer) =>
-                    partygoer.lastSeenIn?.[`${venue.name}/${room.title}`]
-                )
-              : [];
+            const peopleInRoom = recentWorldUsers.filter(
+              (user) => user.lastSeenIn?.[`${venue.name}/${room.title}`]
+            );
```

Comment:
This doesn't have the `lastSeenThreshold` as per https://github.com/sparkletown/internal-sparkle-issues/issues/196#issuecomment-764302115

Reply:
https://github.com/sparkletown/sparkle/pull/1047#discussion_r561862284